caffe2/core/export_caffe2_op_to_c10.h

```diff
@@ -79,7 +80,13 @@ inline void _call_caffe2_op_from_c10(
   // instances in the cache.
   std::vector<IValue> inputs = torch::jit::pop(*stack, num_inputs);
 
-  outputs = (*call_op)(schema, std::move(inputs), std::move(outputs));
+  // Convert outputs to caffe2::Tensor
+  std::vector<caffe2::Tensor> outputs_c2(num_outputs);
+  for (auto i : c10::irange(num_outputs)) {
+    outputs_c2[i] = caffe2::Tensor(outputs.get(i));
+  }
+
+  outputs_c2 = (*call_op)(schema, std::move(inputs), std::move(outputs_c2));
```

Comment:
is it guaranteed that `outputs_c2`'s size afterward is no greater than its size before?

Should we `assert(num_outputs <= outputs_c2.size());`?